apps/volontulo/views/pages.py

```diff
@@ -1,31 +1,34 @@
 # -*- coding: utf-8 -*-
-# pylint: disable=missing-docstring
+
+"""
+.. module:: pages
+"""
+
 from django.core.urlresolvers import reverse_lazy
 from django.shortcuts import redirect
 from django.views.generic.detail import DetailView
-from django.views.generic.edit import (
-    CreateView, UpdateView, DeleteView
-)
+from django.views.generic.edit import CreateView
+from django.views.generic.edit import DeleteView
+from django.views.generic.edit import UpdateView
```

Comment:
Thats not the proper way of doing imports, I suggest using mine version.
E.g: https://github.com/django/django/blob/master/django/db/__init__.py#L2